src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -145,7 +145,7 @@ void BaseQNAcceleration::initialize(
     for (size_t i = 0; i < _dimOffsets.size() - 1; i++) {
       int accumulatedNumberOfUnknowns = 0;
       for (auto &elem : _dataIDs) {
-        auto &offsets = cplData[elem]->mesh->getVertexOffsets();
+        const auto &offsets = cplData[elem]->getVertexOffsets();
```

Comment:
How is this change (the mesh, not the const) related to the PR?

Reply:
`CouplingData::mesh` is private since https://github.com/BenjaminRodenberg/precice/commit/f5c60edb2589544934fc8881c90d3dc1a550ee9e. Not 100% belonging here, but since `mesh` is read-only it does not make sense to expose the complete mesh. We can also cherry pick this change, if you think it should not be part of this PR.

Reply:
This is cleaned up now via 03585b1. Especially 173dbdf.